solr/contrib/ltr/README.md

```diff
@@ -285,17 +284,17 @@ libSVM model format to the format specified above.
     }
 }
 ```
-This is an example of a toy LambdaMART. Type specifies the class to be using to
-interpret the model (LambdaMARTModel in the case of LambdaMART). Name is the
+This is an example of a toy Multiple Additive Trees. Type specifies the class to be using to
+interpret the model (MultipleAdditiveTreesModel in the case of Multiple Additive Trees). Name is the
 model identifier you will use when making request to the ltr framework.
 Features specifies the feature space that you want extracted when using this
 model. All features that appear in the model params will be used for scoring and
 must appear in the features list.  You can add extra features to the features
 list that will be computed but not used in the model for scoring, which can
-be useful for logging. Params are the LambdaMART specific parameters. In this
+be useful for logging. Params are the Multiple Additive Trees specific parameters. In this
 case we have 2 trees, one with 3 leaf nodes and one with 1 leaf node.
 
-A good library for training LambdaMART ( http://sourceforge.net/p/lemur/wiki/RankLib/ ).
+A good library for training LambdaMART, an example of Multiple Additive Trees, is ( http://sourceforge.net/p/lemur/wiki/RankLib/ ).
```

Comment:
I think lambdamart as an example should be mentioned earlier too, maybe replace the link text above from [Multiple Additive Trees] to [LambdaMART] ?